templates/admin/config.tmpl

```diff
@@ -133,7 +133,7 @@
 					<dt>{{.i18n.Tr "admin.config.db_ssl_mode"}}</dt>
 					<dd>{{if .DbCfg.SSLMode}}{{.DbCfg.SSLMode}}{{else}}-{{end}}</dd>
 				{{end}}
-				{{if or (eq .DbCfg.Type "sqlite3") (eq .DbCfg.Type "tidb")}}
+				{{if eq .DbCfg.Type "sqlite3"}}
```

Comment:
else

Reply:
That's legacy old code, I just removed the `tidb`, I think we can leave the logic as it is.